examples/secure-roks-cluster/README.md

```diff
@@ -1,89 +1,104 @@
+|cos_instance_name|Name of the IBM Cloud Object Storage instance. If not provided, an instance is created with the following naming convention: `<var.resource_prefix>-cos`|string|N/A|No|
+|resource_prefix|Prefix to use for created resource names.|string|N/A|Yes|
+|flavor|The flavor for the VPC worker nodes to create in the cluster. To list available flavors, run `ibmcloud ks flavors --zone <vpc_region>-1`.|string|`bx2.4x16`|No|
+|ocp_version|Specify the Red Hat OpenShift on IBM Cloud version. To list versions, run `ibmcloud ks versions`.|string|`4.6.23_1540_openshift`|No|
+|ocp_entitlement|The value that is used to decide how your worker nodes are entitled to run OpenShift Container Platform. For more information, see the [`--entitlement` option description in the docs](https://cloud.ibm.com/docs/openshift?topic=openshift-kubernetes-service-cli#cli_cluster-create-vpc-gen2).|string|N/A|Yes|
+|disable_public_service_endpoint|Disable the public cloud service endpoint to prevent public access to the master.|bool|true|No|
 |worker_nodes_per_zone|The number of worker nodes per zone.|number|3|No|
-|create_timeout|Custom Creation timeout for Cluster|string|N/A|No|
-|roks_kms_policy|Determinies if Kubernetes to KMS Service Authorisation policy.If false it creates policy between services|bool|true|No|
-|kms_instance|Name of Key Protect Instance. If `null` it creates an instance with `<var.resource_prefix>-kp`|string|`null`|No|
-|kms_key|Name of Key Protect Key. If `null` it creates an instance with `<var.resource_prefix>-kp-key`|string|`null`|No|
-|standard_key_type|Determines if it is a standard key or not. This variable is only used during creation of KP Key in this module.|bool|`false`|No|
-|sysdig_name|"Name of Sysdig Instance. If `null` it creates an instance with `<var.resource_prefix>-sysdig`|string|`null`|No|
-|sysdig_access_key|The sysdig monitoring ingestion key that you want to use for your configuration|string|N/A|No|
-|logdna_name|Name of logdna_name Instance. If `null` it creates an instance with `<var.resource_prefix>-logdna`|string|`null`|No|
-|logdna_ingestion_key|The LogDNA ingestion key that you want to use for your configuration|string|N/A|No|
-|private_endpoint|Add this option to connect to your Sysdig and logDNA service instance through the private service endpoint.|bool|true|No|
-|activity_tracker_instance_name|Name of Activity Tracker Instance. If `null` it creates an instance with `<var.resource_prefix>-at`|string|`null`|No|
-|custom_sg_rules|Custom Security rules. Find `custom_sg_rules` object table for more info. |list(object)|[]|No|
-|ip_ranges|Ordered List of ip_ranges on which subnets has to be created. If subnets are created in us-south region, elements in ip_ranges should be equivalent to respective ip range of [`us-south-1`, `us-south-2`, `us-south-3`]|list(string)||Yes|
+|create_timeout|Custom creation timeout for the cluster, in seconds.|string|N/A|No|
```

Comment:
I wanted to verify that this timeout is in seconds.

Reply:
It can be 3s or 3m or 3h (s- seconds, m- minutes, h- hours)
more details https://www.terraform.io/docs/language/resources/syntax.html#operation-timeouts